### ISSUE TITLE
Torsions with a linear angle

### ISSUE THREAD

**Issue by ChayaSt** [lpwgroup/torsiondrive#23]
When `torsiondrive` encounters a torsion where all the rotations have the same energy, it throws errors such as 
```
Cannot continue a constrained optimization; please implement constrained optimization in Cartesian coordinates
```
and 
```
Warning! dihedral values inconsistent with target grid_id
dihedral_values [-150.82428754388542] ref_grid_id (180,)
```
@dgasmith and I submitted a set of molecules. The ones that failed with these failures where torsions that included triple bonds. They were methyl rotors attached to a triple bonded carbon.

I raised an issue on `fragmenter` about flagging such torsions https://github.com/openforcefield/fragmenter/issues/21 and will address it but it would be great if `torsiondrive` can check for it too and give a clearer error message.

**Comment by leeping** [comment]
Hello Chaya,

I understand what you mean.  It makes sense to fail with a better error message if one of the selected dihedral angles contains three linear atoms, because those are not well defined.

Thanks,

- Lee-Ping

**Comment by yudongqiu** [comment]
This is implemented in the recent PR #52. Closing this issue.
